kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/hierarchy/treeview/HierarchyTreeViewController.java

```diff
@@ -85,6 +88,22 @@ protected void initializePanel() {
         // We do not use the platform editing feature because 
         // editing is started on selection + simple click instead of double click
         treeView.setEditable(false);
+        
+        treeView.setOnKeyPressed(event -> {
```

Comment:
Is there any reason for why this is `setOnKeyPressed()` and the other change is `addEventFilter()`?

Reply:
Hello, sorry, I did not receive any mail about your comment. Let me check in the next days.

Reply:
Changed to always call `setOnKerPressed()`

Reply:
Technically this looks okay for me.
But the license header requires an update. It requires the line ` * Copyright (c) 2022 Gluon and/or its affiliates.` as first line.